app/models/plan.rb

```diff
@@ -107,7 +107,7 @@ def guidance_for_question(question)
 		# Get guidance by question where guidance group was selected on creation or if group is organisation default
 		question.guidances.each do |guidance|
 			guidance.guidance_groups.each do |group|
-				if (group.organisation == project.organisation && !group.optional_subset) || project.guidance_groups.include?(group) then
+				if ((group.organisation == project.organisation && !group.optional_subset ) || project.guidance_groups.include?(group) && group.published)  then
```

Comment:
Looks good to me. Being nick-picky here, please remove the additional space before the parenthesis. Thanks!